pandas/core/window.py

```diff
@@ -785,10 +920,52 @@ def validate(self):
         else:
             raise ValueError("Invalid window {0}".format(window))
 
+    def _get_kwargs(self, **kwargs) -> Tuple[Dict, Dict[str, Union[str, Tuple]]]:
```

Comment:
this seems very complicated. can you simplify this whole _get_kwargs bizness?

Reply:
tried to simplify it. it seems ci fails for other reasons

Reply:
What were the reasons? Master was failing due to a pytest dependency issue. That's resolved now.

Reply:
last one is just a network [issue](https://travis-ci.org/pandas-dev/pandas/jobs/569201803#L415)

Reply:
it seems it failed again due to a network [issue](https://travis-ci.org/pandas-dev/pandas/jobs/569677023#L2394)